.mci.yml

```diff
@@ -77,9 +77,10 @@ functions:
                 set -o verbose
                 git clone https://github.com/mongodb/mongo-c-driver.git
                 cd mongo-c-driver
-                git checkout 1.3.4
+                # TODO: Update this to our real minimum for the C++11 driver 3.1 release, once known.
+                git checkout master
                 rm -rf /data/tmp/c-driver-install
-                ./autogen.sh --prefix="/data/tmp/c-driver-install" --enable-tests=no --enable-examples=no --with-libbson=bundled --disable-extra-align --enable-debug --enable-optimizations --disable-shm-counters --disable-static --disable-dependency-tracking --with-pic --disable-automatic-init-and-cleanup ${cdriver_configure_flags}
+                ./autogen.sh --prefix="/data/tmp/c-driver-install" --enable-tests=no --enable-examples=no --with-libbson=bundled --disable-decimal-bid --disable-extra-align --enable-debug --enable-optimizations --disable-shm-counters --disable-static --disable-dependency-tracking --with-pic --disable-automatic-init-and-cleanup ${cdriver_configure_flags}
```

Comment:
This is a test comment to test the new PR review UI.  I guess I click "start a review" now.